### PR TITLE
Update hash and xof function to lower case for consistency with other declared functions

### PR DESCRIPTION
**Pull request by tplooker**
Currently for most functions declared by this spec we are following a syntax of lower case / snake_case name based convention, this PR proposes to change HASH -> hash and XOF -> xof, related discussion in #114

### CODE REVIEW

File: draft-bbs-signatures.md [hunk 1]
```diff
@@ -558,7 +558,7 @@ Procedure:
 
 18. C2 = D * (-r3~) + H_s * s~ + H_j1 * m~_j1 + ... + H_jU * m~_jU
 
-19. c = HASH(PK || Abar || A' || D || C1 || C2 || ph)
+19. c = hash(PK || Abar || A' || D || C1 || C2 || ph)
```
[the remark this answers is not in the document]
Comment (mikelodder7) [reply draft-bbs-signatures.md:561]: This actually should be hash_to_scalar

File: draft-bbs-signatures.md [hunk 2]
```diff
@@ -809,19 +809,19 @@ A cryptographic hash function that takes as an arbitrary octet string input and
 
 - hash\_to\_field\_dst: Domain separation tag used in the hash\_to\_field operation
 
-- hashing\_elements\_to\_scalars: either hash_to_scalar using H (in this case H MUST be an XOF), or hash_to_field with the additional check and re-calculation of more elements until the desired number of non-zero field elements is returned (as described in [Hash to scalar](#hash-to-scalar)).
+- hashing\_elements\_to\_scalars: either hash_to_scalar using hash (in this case hash MUST be an xof), or hash_to_field with the additional check and re-calculation of more elements until the desired number of non-zero field elements is returned (as described in [Hash to scalar](#hash-to-scalar)).
```
Comment (mikelodder7) [inline draft-bbs-signatures.md:812]: not sure what's the difference here between `hash_to_scalar` and `hash_to_field`?

Comment (BasileiosKal) [reply draft-bbs-signatures.md:812]: If I’m not mistaken hash_to_field does not check if the field element is 0. It also does not worry about DST's etc that are not needed for the intended use cases.

File: draft-bbs-signatures.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (BasileiosKal) [reply draft-bbs-signatures.md:561]: I am working on a PR to make this update everywhere it needs to happen

**Review by andrewwhitehead — APPROVED** [review]

File: draft-bbs-signatures.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tplooker) [reply draft-bbs-signatures.md:561]: yes deferring to @BasileiosKal PR that will address this seperately.

**Review by BasileiosKal — APPROVED** [review]

File: draft-bbs-signatures.md [hunk 3]
```diff
@@ -615,15 +615,15 @@ Procedure:
...
+6. domain = OS2IP(hash(PK || L || generators || Ciphersuite_ID || header)) mod q
 
 7. C1 = (Abar - D) * c + A' * e^ + H_s * r2^
 
 8. T = P1 + H_s * domain + H_i1 * msg_i1 + ... H_iR * msg_iR
 
 9. C2 = T * c + D * (-r3^) + H_s * s^ + H_j1 * m^_j1 + ... + H_jU * m^_jU
 
-10. cv = HASH(PK || Abar || A' || D || C1 || C2 || ph)
+10. cv = hash(PK || Abar || A' || D || C1 || C2 || ph)
```
[the remark this answers is not in the document]
Comment (tplooker) [reply draft-bbs-signatures.md:627]: Will be addressed by @BasileiosKal's PR

**Pull request merged**